### PR TITLE
Add stylesheet for generated APIs

### PR DESCRIPTION
**Pull request by lcawl**
This PR is a first draft of a new stylesheet related to auto-generated API output

Relates to https://github.com/elastic/kibana/issues/137240

### CODE REVIEW

File: resources/web/style/openapi.pcss [hunk 1]
```diff
@@ -0,0 +1,182 @@
+#guide {
```
Comment (gtback) [inline resources/web/style/openapi.pcss:1]: I think we should wrap all of this in a `<div class="openapi">` (which would require modifying the HTML template a bit), and nest everything in a `.openapi` block in the PCSS, to reduce the chances that these styles will conflict or override styles defined for the rest of the page.

Comment (gtback) [reply resources/web/style/openapi.pcss:1]: Nevermind, I see you literally just added this 10 seconds before I made my comment 🤣

**Review by gtback — APPROVED** [review]
I just pushed a commit that formatted the PCSS a bit more consistently. 

LGTM!

File: resources/web/style/openapi.pcss (hunk 1 above)
Comment (lcawl) [reply resources/web/style/openapi.pcss:1]: I woke up with that realization this morning, glad we're on the same wavelength!

**Pull request merged**